web-registry/src/components/organisms/CreditBatches.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState, useEffect } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableHead from '@material-ui/core/TableHead';
+import axios from 'axios';
+import moment from 'moment';
+import cx from 'clsx';
+
+import {
+  StyledTableContainer,
+  StyledTableCell,
+  StyledTableRow,
+  StyledTableSortLabel,
+} from 'web-components/lib/components/table';
+import { getComparator, stableSort, Order } from 'web-components/lib/components/table/sort';
+import Section from 'web-components/lib/components/section';
+import { truncateWalletAddress } from '../../lib/wallet';
+import { ledgerRestUri } from '../../ledger';
+
+interface BatchRowData {
+  start_date: string | Date;
+  end_date: string | Date;
+  issuer: string;
+  batch_denom: string;
+  class_id: string;
+  total_amount: number;
+  amount_cancelled: number;
```

Comment:
this is not the amount cancelled but amount retired

Reply:
added amount_retired and amount_active